src/utils/constants.ts

```diff
@@ -174,6 +174,8 @@ export const DEFAULT_CDD_ID = '0x00000000000000000000000000000000000000000000000
  */
 export const MINIMUM_SQ_VERSION = '12.2.0-alpha.2';
 
+export const SETTLEMENTS_V2_SQ_VERSION = '16.0.0-alpha.1';
```

Comment:
Should these versions only reference official releases? I think they do show to the user if they have a bad version.

I get why we'd need `alpha.1` for testing in dev, but it should be OK to make this a plain 16 as long as we coordinate the release timing properly

Reply:
But coordinating and updating on the right time can be tricky. It's OK to leave as is with this to be removed soon. And also, this is not getting logged anywhere. Its just for checking, so anyone using middleware alpha version would also get the correct answer to the queries

Reply:
I thought this was replacing minimum SQ version 😅